web-client/iron-remote-desktop/src/iron-remote-desktop.svelte

```diff
@@ -129,18 +132,24 @@
         return (evt.ctrlKey && evt.code === 'KeyV') || evt.code == 'Paste';
     }
 
+    function isContent(content: ClipboardContent) {
+        // Check whether function exists. To make it more robust we can check every method.
+        return content.mime_type() !== undefined || content.value() !== undefined;
+    }
```

Comment:
thought: It looks like we are checking whether a value implementing the `ClipboardContent` interface, is implementing the `ClipboardContent` interface :thinking: